models/docs/contributing.rst

```diff
@@ -18,3 +18,14 @@ anemoi-models include unit tests that can be executed locally using
 pytest. For more information on testing, please refer to the
 :ref:`general Anemoi testing guidelines
 <anemoi-docs:testing-guidelines>`.
+
+*******************************
+ Provide checkpoint Migrations
+*******************************
+
+If your changes break existing checkpoints, you must provide a
```

Comment:
Could be worth mentioning here how one can detect if the changes break the checkpoint - ie mention the tests that  checks that?

Reply:
I don't think it's documented but Vera can confirm. Adding a quick mention to this test would be a nice addition.